internal/events/event_dispatcher.go

```diff
@@ -151,7 +151,7 @@ func (ed *eventDispatcher) electAndStart() {
 	<-ed.eventPoller.closed
 }
 
-func (ed *eventDispatcher) getEvents(ctx context.Context, filter database.Filter) ([]fftypes.LocallySequenced, error) {
+func (ed *eventDispatcher) getEvents(ctx context.Context, filter database.Filter, offset int64) ([]fftypes.LocallySequenced, error) {
```

Comment:
Doesn't look like this param is used?

Reply:
Was added for logging on the the common interface with pins - will add similar logging here